lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_RebadgePercentilesAsMembers.py

```diff
@@ -114,6 +115,18 @@ def test_still_works_if_percentile_coord_is_different(self):
         self.assertArrayAlmostEqual(
             result.coord("realization").points, np.array([0, 1, 2]))
 
+    def test_raises_exception_if_realization_already_exists(self):
+        """
```

Comment:
Good point. I've changed this and changed all existing ones in this file to use the same style.
(Note: other docstrings still contain references to 'members', but as we've discussed, this can be addressed in a follow-on ticket).